www/_includes/header.html

```diff
@@ -29,18 +29,24 @@
                         <li>
                             <a href="{{ site.baseurl }}/#getstarted">Get Started</a>
                         </li>
+                        <li>
+                            <form class="navbar-form navbar-right" id="header-search-form" role="search">
+                                <div class="input-group">
+                                {% if page.version %}
```

Comment:
Nitpick: please indent this block; it looks weird indented one level lower than the code below it.